store/tikv/gcworker/misc.go

```diff
@@ -50,11 +50,20 @@ var (
 			Name:      "gc_failure",
 			Help:      "Counter of gc job failure.",
 		}, []string{"type"})
+
+	gcRegionTooMuchLocksCounter = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Namespace: "tidb",
+			Subsystem: "tikvclient",
+			Name:      "gc_region_too_much_locks",
+			Help:      "Counter of gc scan lock req more than once in the same region.",
```

Comment:
req means require or request?

Reply:
It means request, but I think I should make it clear here.